src/hardware/eeprom.rs

```diff
@@ -1,10 +1,6 @@
 use embedded_hal::blocking::{delay::DelayMs, i2c::WriteRead};
 
-// The EEPROM is a variant without address bits, so the 3 LSB of this word are "dont-cares".
-const I2C_ADDR: u8 = 0x50;
```

Comment:
make them `pub` here and use `hardware::eeprom::I2C_ADDR` etc in your code.
No need to move them into `hardware`.